.circleci/config/jobs/operator_upgrade_tests.yml

```diff
@@ -14,30 +14,31 @@ steps:
   - run:
       name: Install required packages
       command: |
-        sudo apt install -y uuid-runtime
+        sudo apt update
+        sudo apt install -y uuid-runtime make
         python -m pip install requests pyyaml
-        python -m pip install operator-courier==2.1.7
+        python scripts/operator/download_operator_sdk.py
+        OPM_VERSION=v1.13.0 python scripts/operator/download_operator_package_manager.py
```

Comment:
Why not make it as an env variable? I see here it's defind as `v1.13.0` and it scripts in try/except block it's `v.1.16.1`. Is this how it should be?

Reply:
The machine executor uses an old debian version, the latest OPM 1.16.1 requires a later version of GLIBC that the current machine cannot have :/

Reply:
Added a comment there